src/app/charts/extra-params-components/extra-params.constants.ts

```diff
@@ -26,3 +31,12 @@ export function hasExtraParams(indicatorName: string): boolean {
     }
     return false;
 }
+
+export function isPercentileIndicator(indicatorName: string): boolean {
+    for (const percentileName of percentileIndicatorNames) {
```

Comment:
Mostly trivial comment, but is there a reason these functions can't all be compressed down to the more commonly used:
```
export function isPercentileIndicator(indicatorName: string): boolean {
    return percentileIndicatorNames.indexOf(indicatorName) !== -1;
}
```